apps/docs/content/zerops-yml/specification.mdx

```diff
@@ -135,6 +135,10 @@ build:
 
```

Comment:
based on the discord thread, I would also add the exact example from it here (sth like :point_down: )

that e.g. `./src/assets/fonts` would result in `/var/www/src/assets/fonts` in runtime

just to make sure people find it